python/simple/v1/app.py

```diff
@@ -0,0 +1,63 @@
+from ampli import *
+from plugins import MyEventIDPlugin
+
+# Initialize the Ampli instance with LoadOptions and LoadClientOptions
+ampli = Ampli()
+ampli.load(LoadOptions(Environment.DEVELOPMENT, False, client=LoadClientOptions(api_key="key", zone="EU")))
```

Comment:
Can we import this key from a `.env` file or equivalent to make it easier to configure without commiting keys.
https://github.com/amplitude/ampli-examples/blob/main/node/typescript/src/app.ts#L14